.github/workflows/default_plugins.yml

```diff
@@ -2,7 +2,7 @@ name: Publish Default Plugins
 
 on:
   push:
-    branches: ['dev']
+    branches: ['master']
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.

Reply:
Try now

Reply:
Provide me a very detailed comprehensive end-user friendly summary of the changes in this PR's description, categorised in new features and bug fixes.